layouts/shortcodes/featured/scrolly.html

```diff
@@ -1,13 +1,53 @@
 <div
-  x-data="{ activeIndex: -1 }"
-  class="scrolly-container scroll-snap-y scroll-snap-mandatory bg-black text-white"
+  x-data="{
+   activeIndex: -1,
+   buttonPosition: 'fixed',
+   
+   updateButtonPosition() {
+     const container = this.$refs.container;
+     const button = this.$refs.skipButton;
+     
+     if (!container || !button) return;
+     
+     const containerRect = container.getBoundingClientRect();
+     const viewportHeight = window.innerHeight;
+     
+     if (containerRect.bottom < viewportHeight - 6) {
+       this.buttonPosition = 'absolute';
+     } else {
+       this.buttonPosition = 'fixed';
+     }
+   }
+ }"
+  x-init="
+   $nextTick(() => {
+     updateButtonPosition();
+     window.addEventListener('scroll', () => updateButtonPosition());
+     window.addEventListener('resize', () => updateButtonPosition());
```

Comment:
These need to be debounced. Otherwise you're calculating BoundingClientRect for every sub-pixel of scrolling and it slows the computer down too much.

Reply:
You could do it by using `@scroll.window.debounce="updateButtonPosition()"` instead, I think.